src/components/Sections/SectionDuration.js

```diff
@@ -57,6 +58,13 @@ const SectionDuration = ({ data, style, chartProperties, title, titleStyle, forc
   const format = chartProperties && chartProperties.format;
   const labels = getLabels(chartProperties);
   const parts = [];
+  const containerStyle = cloneDeep(style);
+  let color;
+
+  if (containerStyle) {
+    color = containerStyle.backgroundColor;
```

Comment:
what if the style object contains color?

Reply:
Good point, it will apply the style color to the container as expected.
In general, the background color is generated in the server as part of the threshold functionality.
As part of the new requirements the coloring section changed to icon only instead of background.
The color variable used for icon only.